frontend/app/src/services/rotkehlchen-api.ts

```diff
@@ -688,6 +705,18 @@ export class RotkehlchenApi {
     return request.then(handleResponse);
   }
 
+  listReports(page?: string, rows?: string): Promise<boolean> {
+    return this.axios
+      .get<ActionResult<boolean>>('/reports/', {
+        params: {
+          page: page,
+          rows: rows
+        },
```

Comment:
As we discussed instead of page and rows it would be make sense to move to `limit` & `offset`